Code/Engine/RendererCore/Debug/Implementation/DebugRenderer.cpp

```diff
@@ -983,14 +1067,66 @@ void ezDebugRenderer::RenderInternal(const ezDebugRendererContext& context, cons
     return;
   }
 
-  pDoubleBufferedContextData->m_uiLastRenderedFrame = ezRenderWorld::GetFrameCounter();
-
   PerContextData* pData = pDoubleBufferedContextData->m_pData[ezRenderWorld::GetDataIndexForRendering()].Borrow();
   if (pData == nullptr)
   {
     return;
   }
 
+  // update the frame counter
+  pDoubleBufferedContextData->m_uiLastRenderedFrame = ezRenderWorld::GetFrameCounter();
+
+  // draw info text
```

Comment:
This is still not the right place. This block ("draw info text") has to be before line 1064. Just like the presistent stuff.

Reply:
Yep, my previous change was wrong.